packages/components/src/templates/next/components/internal/GoogleTagManager/utils.ts

```diff
@@ -0,0 +1,10 @@
+export const shouldNotIncludeGoogleTagManager = (): boolean => {
+  if (typeof process === "undefined") return true
+  if (process.env.NODE_ENV !== "production") return true
+  return false
+}
+
+export const getIsomerGoogleTagManagerId = (): string | undefined => {
+  if (shouldNotIncludeGoogleTagManager()) return undefined
+  return process.env.NEXT_PUBLIC_ISOMER_GOOGLE_TAG_MANAGER_ID
```

Comment:
actually i will just update this to `undefined` for now - this is better to hardcode it in once we have our own GTM. Do you have access to your google account? if not i will wait for Jan to be back to check with him

Reply:
Hmm I was thinking otherwise. I think this one shouldn't be hard coded especially now our codebase is being used by other people, so at least it would be easier for them to swap out this value with their own GTM IDs.

Reply:
fixed in https://github.com/opengovsg/isomer/pull/894/commits/1c3064924ed51c61db6bab2fede89cf59db50ac4 and have it the GTM ID be passed in during codebuild https://github.com/opengovsg/isomer-next-infra/pull/34